include/seqan3/io/alignment_file/sam_tag_dictionary.hpp

```diff
@@ -25,16 +25,16 @@ namespace seqan3::detail
 {
     //!\brief std::variant of allowed types for optional tag fields of the SAM format.
     //!\ingroup alignment_file_io
-    using sam_tag_variant = std::variant<char, int32_t, float, std::string,
+    using sam_tag_variant = std::variant<char, int32_t, float, std::string, std::vector<std::byte>,
```

Comment:
usually we wrap all or nothing. I see the pattern for int8_t and uint8_t, but at least wrap std::string and std::vector<byte> on a new line each